lib/paperclip-meta/attachment.rb

```diff
@@ -18,12 +24,7 @@ def save_with_meta_data
 
         def post_process_styles_with_meta_data(*styles)
```

Comment:
`post_process` is not called until the background job is being run when using `delayed_paperclip`